cloud-watch-to-slack-testing/deployment/index.js

```diff
@@ -291,12 +291,19 @@ function setSlackChannelBasedOnSNSTopic(topicArn) {
 
 function processEvent(event, callback) {
   console.log(event);
-  const message = JSON.parse(event.Records[0].Sns.Message);
+  let message;
+  let messageText;
+  try {
+    message = JSON.parse(event.Records[0].Sns.Message);
+    messageText = messageTextFromMessageObject(message);
+  } catch (e) {
+    message = event.Records[0].Sns.Message;
+    messageText = message;
```

Comment:
This seems like a good spot to log that the message was handled within the catch vs silently passing through.

Reply:
Added